grafana/dashboards/cmode/volume.json

```diff
@@ -164,7 +164,7 @@
       ],
       "timeFrom": null,
       "timeShift": null,
-      "title": "Volume Average Latency",
+      "title": "Top $TopResources Volumes by Average Latency",
```

Comment:
it's stat panel which show one value of avg of topN, title may not reflect that.